### DISCUSSION TITLE
Video recording resolution and FPS

### DISCUSSION THREAD

**Discussion by benatwalkly** [Apparence-io/CamerAwesome#249] [Q&A]
Hello,
Is there a way to set the values for FPS and image resolution for recording video similar to what you can do in the ios camera app (screenshot) for example?
![image](https://user-images.githubusercontent.com/125352404/220640077-30327331-7b77-4aa3-9312-fc925e342f36.png)

Kind regards,
Ben

**Comment by apalala-dev** [comment]
Hey @benatwalkly 
Unfortunately, this feature is not yet available in CamerAwesome.
We'll let you know when we implement it!

**Reply by benatwalkly** [reply]
Thank you for quick reply. 
Do you have any insights on a timeline for this? 
Or do you have some visibility on a roadmap for CamerAwesome?

**Reply by apalala-dev** [reply]
No ETA yet, we already have some other work that we'd like to finish and merge, as you can see on the list of our PRs.
This particular change (at least the resolution) should not be too hard to implement if you want to have a look yourself and submit your PR :)
